src/components/approval-type/ApprovalType.vue

```diff
@@ -0,0 +1,157 @@
+<template>
+  <div id="approval-type">
+    <v-row no-gutters>
+      <v-col cols="12" sm="3" class="pr-4">
+        <label id="approval-type-label">Approval Type</label>
+      </v-col>
+      <v-col cols="12" sm="9">
+        <v-radio-group class="payment-group" v-model="approvalTypeSelected" @change="radioButtonChanged">
+          <!-- COURT ORDER radio button -->
+          <v-radio id="court-order-radio" class="mb-0"
+             :label="getRadioText(ApprovalTypes.VIA_COURT_ORDER)"
+             :value="ApprovalTypes.VIA_COURT_ORDER"
+          />
+          <v-form ref="courtNumRef" id="court-num-form" v-model="valid" class="mt-4 ml-8">
+            <v-text-field
+              @input="courtOrderNumberChanged"
+              id="court-order-number-input"
+              v-model="courtOrderNumber"
+              label="Court Order Number"
+              :rules="courtOrderNumRules"
+              :disabled="approvalTypeSelected === ApprovalTypes.VIA_REGISTRAR"
+              @update:error="emitValidationError($event)"
+              filled
+            />
+          </v-form>
+          <!-- REGISTRAR radio button -->
+          <v-radio id="registrar-radio" class="mb-0 pt-2"
+             :label="getRadioText(ApprovalTypes.VIA_REGISTRAR)"
+             :value="ApprovalTypes.VIA_REGISTRAR"
+             v-if="!isCourtOrderOnly"
```

Comment:
similar  to the above comment, leading with a conditional (when present) is what we've done in the past, with regards to arrangement of the component props. 
ie `<v-radio v-if="...`

Reply:
Makes sense. I'll fix it.